### DISCUSSION TITLE
Openstack images

### DISCUSSION THREAD

**Discussion by JurriaanThinkOpen** [burmilla/os#166] [General]
The documentation mentions that there are QCOW3 images, but I do not see them under the releases. These were available for the RancherOS releases.

Are these still being built? Anyone running BurmillaOS on Openstack?

**Comment by olljanat** [comment]
You can create Openstack image with https://github.com/burmilla/os-packer

**Reply by JurriaanThinkOpen** [reply]
Thanks for the hint.

I couldn't get the packer working on my windows host due to the qemu dependencies, but the scripts were enough to allow me to create a working image with virtualbox.

For if anyone is wondering, the image is simply a ros install to disk with some specific startup options and can be found under [scripts/install2disk.](https://github.com/burmilla/os-packer/blob/824681b4b37fda73e5da6603bdcf98948234c586/scripts/install2disk) The same for the other builds.
